sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/pubsub/PubsubUnboundedSink.java

```diff
@@ -236,8 +237,8 @@ public void populateDisplayData(Builder builder) {
     private transient PubsubClient pubsubClient;
 
     private final Counter batchCounter = Metrics.counter(WriterFn.class, "batches");
-    private final Counter elementCounter = Metrics.counter(WriterFn.class, "elements");
-    private final Counter byteCounter = Metrics.counter(WriterFn.class, "bytes");
+    private final Counter elementCounter = IOMetrics.elementsWritten();
```

Comment:
I wasn't aware these metrics were also on the table.
So we should probably have better namespaces to differentiate the metrics you mentioned from IO metrics.
If the term `io` as a namespace may cause confusion, perhaps we can use `source` and `sink` as namespaces for the IO metrics instead.